homeassistant/core.py

```diff
@@ -305,7 +302,7 @@ class HassJobType(enum.Enum):
     Executor = 3
 
 
-class HassJob(Generic[_P, _R_co]):
+class HassJob[**_P, _R_co]:
```

Comment:
The `_co` suffix isn't required anymore since the variance is inferred. However, left it in place to help with understanding for what the TypeVar is used (and how it should be inferred).